packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -205,6 +232,41 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
                 this.activateTerminal(termWidget);
             }
         }));
+        commands.registerCommand(TerminalCommands.SCROLL_LINE_UP, {
+                isEnabled: () => this.shell.activeWidget instanceof TerminalWidget,
+                isVisible: () => false,
+                execute: () => {
+                    (this.shell.activeWidget as TerminalWidget).scrollLineUp();
+                }
+        });
+        commands.registerCommand(TerminalCommands.SCROLL_LINE_DOWN, {
+            isEnabled: () => this.shell.activeWidget instanceof TerminalWidget,
+            isVisible: () => false,
+            execute: () => {
+                (this.shell.activeWidget as TerminalWidget).scrollLineDown();
+            }
+        });
+        commands.registerCommand(TerminalCommands.SCROLL_TO_TOP, {
```

Comment:
@AndrienkoAleksandr are we missing the keybinding for the opposite `Scroll To Bottom`?
It looks like it works by using <kbd>End</kbd>

Reply:
Hello. I don't know how, but it already works by `End`... Strange... That's why I didn't apply this command.

Reply:
> Hello. I don't know how, but it already works by `End`... Strange... That's why I didn't apply this command.

That's what I noticed as well, <kbd>End</kbd> already worked correctly, perhaps we can omit it.

Reply:
Agree, I think we can omit it. If somebody will report issue, we can fix it easily.